python_modules/libraries/dagster-pyspark/dagster_pyspark_tests/test_types.py

```diff
@@ -12,14 +12,20 @@
 
 spark = SparkSession.builder.getOrCreate()
 
-dataframe_parametrize_argnames = "file_type,read,other"
+dataframe_parametrize_argnames = "file_type,read,other,resource"
 dataframe_parametrize_argvalues = [
-    pytest.param("csv", spark.read.csv, False, id="csv"),
-    pytest.param("parquet", spark.read.parquet, False, id="parquet"),
-    pytest.param("json", spark.read.json, False, id="json"),
-    pytest.param("csv", spark.read.load, True, id="other_csv"),
-    pytest.param("parquet", spark.read.load, True, id="other_parquet"),
-    pytest.param("json", spark.read.load, True, id="other_json"),
+    pytest.param("csv", spark.read.csv, False, pyspark_resource, id="csv"),
+    pytest.param("parquet", spark.read.parquet, False, pyspark_resource, id="parquet"),
+    pytest.param("json", spark.read.json, False, pyspark_resource, id="json"),
+    pytest.param("csv", spark.read.load, True, pyspark_resource, id="other_csv"),
+    pytest.param("parquet", spark.read.load, True, pyspark_resource, id="other_parquet"),
+    pytest.param("json", spark.read.load, True, pyspark_resource, id="other_json"),
+    pytest.param("csv", spark.read.csv, False, lazy_pyspark_resource, id="csv"),
```

Comment:
Nit: Mind if we update these `id` fields so that they are different for the regular pyspark resource versus the lazy pyspark resources? e.g. `pyspark_csv`, `lazy_pyspark_csv`...

Reply:
sure thing that makes sense! I'll make that change